commons/src/main/java/io/hyscale/commons/component/ExecutableComponentProvider.java

```diff
@@ -18,7 +18,7 @@
 import javax.annotation.PreDestroy;
 import java.util.Collections;
 import java.util.List;
-
+@SuppressWarnings("java:S3740")
```

Comment:
Use the generic reference where required in this class. 
Like List<InvokerHook<C>> instead of List<InvokerHook> enables strict type checking.
Also , InvokerHook<C> instead of InvokerHook